src/v2/spec.yaml

```diff
@@ -31,6 +31,12 @@ components:
       enum:
         - like
         - recast
+    GetReactionTypes:
```

Comment:
`ReactionTypesFilter` maybe? to go better with the ReactionType enum above it?

Reply:
named it `ReactionsTypes`